src/widgets/checkbox.cpp

```diff
@@ -198,7 +198,7 @@ namespace gcn
 
     void CheckBox::adjustSize()
     {
-        int height = getFont()->getHeight();
+        const int height = getFont()->getHeight() + 2;
```

Comment:
I dislike to have that change (`+ 2`) in a PR which clean `const` and `std::move`.

Reply:
you're right, that shouldn't be part of it. I'll fix that.